src/utilities/browser-utils.test.ts

```diff
@@ -0,0 +1,55 @@
+import { BrowserUtils } from "./browser-utils";
+import { JSDom } from "jsdom";
+
+describe("BrowserUtils", () => {
+  describe("isIE", () => {
+    it("when wondow.document.documentMode is undefined, then returns false", () => {
+      // Arrange
+      window.document.documentMode = undefined;
+
+      // Act
+      const result = BrowserUtils.isIE();
+
+      // Assert
+      expect(result).toBeFalse();
+    });
+
+    it("when window.document.documentMode is truthy, then returns true", () => {
+      // Arrange
+      window.document.documentMode = true;
+
+      // Act
+      const result = BrowserUtils.isIE();
+
+      // Assert
+      expect(result).toBeTrue();
+    });
+  });
+
+  describe("isNotIE", () => {
+    it("when wondow.document.documentMode is undefined, then returns true", () => {
+      // Arrange
+      window.document.documentMode = undefined;
+
+      // Act
+      const result = BrowserUtils.isNotIE();
+
+      // Assert
+      expect(result).toBeTrue();
+    });
+
+    it("when window.document.documentMode is truthy, then returns false", () => {
+      // Arrange
+      window.document.documentMode = true;
+
+      // Act
+      const result = BrowserUtils.isNotIE();
+
+      // Assert
+      expect(result).toBeFalse();
+    });
+  });
+
+  // Clean up
+  window.document.documentMode = undefined;

```

Comment:
Do you want this after each test? If so use 'afterEach` and put at top of current describe block.

Reply:
@wintondeshong each test is setting the value to whatever it needs to be in it's `// Arrange` section.

Reply:
@wintondeshong each test is setting the value to whatever it needs to be in it's `// Arrange` section.

Reply:
@mrjones2014 yeah, then you'll need to call `` in an afterEach in the top-level describe. I'll merge and make the change

```javascript
afterEach(() => {
    window.document.documentMode = undefined;
});
```